<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentForm</class>
 <widget class="QDialog" name="DocumentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>314</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QFrame" name="frame">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <widget class="QFrame" name="left_frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="adddocs_frame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>70</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="margin">
              <number>5</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="naklad_header_label">
               <property name="text">
                <string>Накладная</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="naklad_label">
               <property name="text">
                <string>№ от</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="sf_header_label">
               <property name="text">
                <string>Счёт-фактура</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="sf_label">
               <property name="text">
                <string>№ от</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="provider_box">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Поставщик</string>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>421</width>
               <height>27</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="provider_label">
                <property name="text">
                 <string>Фирма:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="DoubleClickLineEdit" name="provider_edit">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="provider_button">
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>X</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QGroupBox" name="shipper_box_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>60</y>
               <width>437</width>
               <height>91</height>
              </rect>
             </property>
             <property name="title">
              <string>Покупатель</string>
             </property>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>421</width>
                <height>58</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="contragent_label">
                 <property name="text">
                  <string>Контрагент:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="DoubleClickLineEdit" name="contragent_edit">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="contragent_button">
                 <property name="maximumSize">
                  <size>
                   <width>25</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="cons_label">
                 <property name="text">
                  <string>Грузополучатель:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="DoubleClickLineEdit" name="cons_edit">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="cons_button">
                 <property name="maximumSize">
                  <size>
                   <width>25</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="right_frame">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="nds_label">
                <property name="text">
                 <string>НДС</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="nds_edit"/>
              </item>
              <item>
               <widget class="QLabel" name="percents_label">
                <property name="text">
                 <string>%</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="nds_button">
              <property name="text">
               <string>Пересчитать</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="tempdocs_box">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="title">
             <string>Изменить</string>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>131</width>
               <height>61</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QComboBox" name="convert_combobox">
                <property name="maximumSize">
                 <size>
                  <width>120</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QPushButton" name="s_button">
                  <property name="maximumSize">
                   <size>
                    <width>25</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>С</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="n_button">
                  <property name="maximumSize">
                   <size>
                    <width>25</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Н</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="sf_button">
                  <property name="maximumSize">
                   <size>
                    <width>25</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>СФ</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="weight_box">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>70</height>
             </size>
            </property>
            <property name="title">
             <string>Вес</string>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>20</y>
               <width>135</width>
               <height>43</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="weight_label">
                <property name="text">
                 <string>Информация неполная!</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="weight_value_label">
                  <property name="text">
                   <string>0.00</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="kg_label">
                  <property name="text">
                   <string>кг</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="notes_frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="note_label">
         <property name="text">
          <string>Комментарий:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="note_edit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="who_label">
         <property name="text">
          <string>Для кого счёт:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="RComboBox" name="r_combobox">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QCheckBox" name="print_checkbox">
         <property name="text">
          <string>Печать</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="add_rstring_button">
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleClickLineEdit</class>
   <extends>QLineEdit</extends>
   <header>common/helpers.h</header>
  </customwidget>
  <customwidget>
   <class>RComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">docs/rcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
